src/import/FSHTank.ts

```diff
@@ -179,6 +203,41 @@ export class FSHTank implements Fishable {
               e.id === item ||
               getUrlFromFshDefinition(e, this.config.canonical) === item
           );
+          if (!result) {
+            // There may be a matching definitional Instance of StructureDefinition with type Extension
+            result = this.getAllInstances().find(
+              extensionInstance =>
+                (extensionInstance.name === item ||
+                  extensionInstance.id === item ||
+                  getUrlFromFshDefinition(extensionInstance, this.config.canonical) === item) &&
+                extensionInstance.instanceOf === 'StructureDefinition' &&
+                extensionInstance.usage === 'Definition' &&
+                extensionInstance.rules.some(
+                  rule =>
+                    rule instanceof AssignmentRule &&
+                    rule.path === 'derivation' &&
+                    rule.value instanceof FshCode &&
+                    rule.value.code === 'constraint'
+                ) &&
+                extensionInstance.rules.some(
+                  rule =>
+                    rule instanceof AssignmentRule &&
+                    rule.path === 'type' &&
+                    rule.value === 'Extension'
+                )
+            );
+          }
+          if (!result) {
```

Comment:
I remember talking about this and it making more sense at the time, but now I am getting confused. We are searching for an `Extension`, which is to say, a `StructureDefinition` of type `Extension`. I don't think we would want to return something which is an `InstanceOf` `Extension`, since that is not going to be an SD.

Reply:
I think I'm in the same situation as you: it made sense before, but thinking about it again, I don't think we want `InstanceOf` `Extension`. That instance would be a specific element, not a definition of the rules for an extension. I'll change this implementation accordingly.